### PR TITLE
Add a v2 client library, starting with machine information.

### PR DESCRIPTION
**Pull request by rjnagal**
I'll probably move the existing API down to a v1 directory next.

### CODE REVIEW

File: client/v2/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```go
+client.VersionInfo()
+```
+
+This method returns the cAdvisor version.
+
+### Attributes
+
+```go
+client.VersionInfo()
```
Comment (vmarmol) [inline client/v2/README.md]: s/VersionInfo/Attributes/

File: client/v2/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+This method returns a cadvisor/info/v2/Attributes struct with all the fields filled in. Attributes includes hardware attributes (as returned by MachineInfo) as well as software attributes (eg. software versions). Here is an example return value:
```
Comment (vmarmol) [inline client/v2/README.md]: link to v2 file for cadvisor/info/v2/Attributes?

File: client/v2/client.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+func (self *Client) versionInfoUrl() string {
+	return self.baseUrl + path.Join("version")
+}
+
+func (self *Client) attributesUrl() string {
+	return self.baseUrl + path.Join("attributes")
+}
+
+func (self *Client) httpGetResponse(data, postData interface{}, url, infoName string) ([]byte, error) {
```
Comment (vmarmol) [inline client/v2/client.go]: Looks like data is never used

File: client/v2/README.md (hunk 1 above)
Comment (rjnagal) [inline client/v2/README.md]: done

File: client/v2/README.md (hunk 2 above)
Comment (rjnagal) [inline client/v2/README.md]: added

File: client/v2/client.go (hunk 3 above)
Comment (rjnagal) [inline client/v2/client.go]: fixed.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**